### DISCUSSION TITLE
Deepgram free transcription tool doesn't work

### DISCUSSION THREAD

**Discussion by Farooq87** [deepgram/community#974] [General help] [answered]
Please fix this useful tool. When I hit 'record' button nothing happens. App is seemingly out of order.

https://deepgram.com/free-transcription?ref=deepgram&utm_source=deepgram&utm_medium=referral

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Comment by Farooq87** [comment]
@srisch
@team-deepgram 
Could you please look at this issue?

**Comment by jpvajda** [comment, marked as answer]
@Farooq87  Thanks for letting us know, i'm also getting 500 errors when trying to run transcriptions, so let me let follow up in this with a few people.

**Comment by Farooq87** [comment]
@jpvajda
@team-deepgram
Sorry, guys
It still doesn't work for me

[28.10.2024, 11_56_04 - Screen - Untitled video.webm](https://github.com/user-attachments/assets/e6d39003-4ec7-4710-81cc-772bf8023538)

**Comment by deepgram-community** [comment]
The issue has not been resolved yet, but a team is working on it. I currently have no ETA on a fix, but I'lll re-open this question

> ![avatar](https://avatars.slack-edge.com/2023-02-20/4855338233600_fad9174023ee3f640ce1_32.png) This message was sent by John Vajda from Deepgram, via our community automation.

**Comment by Farooq87** [comment]
Thank you, Deepgram team!
